hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_hasher.py

```diff
@@ -42,21 +42,18 @@ def get_image_bytes(
     Once we have more hashing lambdas that need access to media, this could be
     moved into its own module.
     """
-    if isinstance(submission_message, URLImageSubmissionMessage):
-        return URLImageSource().get_image_bytes(submission_message.url)
-    else:
-        return S3BucketImageSource(
-            submission_message.bucket, default_s3_bucket_image_prefix
-        ).get_image_bytes(submission_message.content_id)
+    return S3BucketContentSource(
+        submission_message.bucket, default_s3_bucket_image_prefix
+    ).get_bytes(submission_message.content_id)
```

Comment:
See above. Why not move this to hashing.py as well?

Reply:
Answered above. Just reducing surface area of change until I have proven out matcher.

Right now, hashing.py just does the hashing. It does not forward that to any other system. Once it starts to do that, I'll connect these pieces.